lasy/profiles/transverse/hermite_gaussian_profile.py

```diff
@@ -10,31 +10,67 @@ class HermiteGaussianTransverseProfile(TransverseProfile):
     r"""
     A high-order Gaussian laser pulse expressed in the Hermite-Gaussian formalism.
 
+    Definition is according to Siegman "Lasers" pg. 646 eq. 60, as explicitly given
+    in https://doi.org/10.1364/JOSAB.489884 eq. 3, with the beam center upon the
+    optical axis, :math:`x_0,y_0 = (0,0)`.
+
     More precisely, the transverse envelope (to be used in the
     :class:`.CombinedLongitudinalTransverseLaser` class) corresponds to:
 
     .. math::
-
         \mathcal{T}(x, y) = \,
-        \sqrt{\frac{2}{\pi}} \sqrt{\frac{1}{2^{n} n! w_0}}\,
-        \sqrt{\frac{1}{2^{n} n! w_0}}\,
-        H_{n_x}\left ( \frac{\sqrt{2} x}{w_0}\right )\,
-        H_{n_y}\left ( \frac{\sqrt{2} y}{w_0}\right )\,
-        \exp\left( -\frac{x^2+y^2}{w_0^2} \right)
+                \mathcal{H}_m (x) \, \mathcal{H}_n(y) \, \exp(i \Phi)
+
+    with
+
+    .. math::
+        \mathcal{H}_m(x) = A_m h_m \left ( \frac{\sqrt{2}x}{w_x(z)} \right) \exp{\left( -\frac{x^2}{w_x^2(z)}\right)} \exp{ \left ( -i k_0 \frac{x^2}{2 R_x(z)} \right )}
+
+        \mathcal{H}_n(y) = A_n h_n \left ( \frac{\sqrt{2}y}{w_y(z)} \right) \exp{\left( -\frac{y^2}{w_y^2(z)}\right)} \exp{ \left ( -i k_0 \frac{y^2}{2 R_y(z)} \right )}
+
```

Comment:
Instead of writing everything twice for $\mathcal{H}_m(x)$ **and** $\mathcal{H}_n(y)$, I would recommend to keep the equation line 21-22 as is but write the rest $\mathcal{H}_p(q)$ only once below, to avoid duplication.
```suggestion
        \mathcal{H}_p(q) = A_p h_p \left ( \frac{\sqrt{2}q}{w_q(z)} \right) \exp{\left( -\frac{q^2}{w_q^2(z)}\right)} \exp{ \left ( -i k_0 \frac{q^2}{2 R_q(z)} \right )}

```